dataactcore/migrations/versions/ff4728a82180_add_duns_table.py

```diff
@@ -33,7 +33,7 @@ def upgrade_data_broker():
     sa.Column('created_at', sa.DateTime(), nullable=True),
     sa.Column('updated_at', sa.DateTime(), nullable=True),
     sa.Column('duns_id', sa.Integer(), nullable=False),
-    sa.Column('awardee_or_recipient_uniqu', sa.Text(), nullable=True),
+    sa.Column('awardee_or_recipient_uniqu', sa.Text(), nullable=True, index=True),
```

Comment:
I think you still have to do the `op.create_index('ix_duns_awardee_or_recipient_uniqu', 'duns', ['awardee_or_recipient_uniqu'], unique=False)` thing. I'm not sure this works this way (we don't have it like this in any of our other migrations)